sotodlib/site_pipeline/make_book.py

```diff
@@ -1,13 +1,42 @@
 import os
 import traceback
 import argparse
+
 import datetime as dt
 from typing import Optional
-from sotodlib.io.imprinter import Imprinter
+from sotodlib.io.imprinter import Imprinter, Books, FAILED
 import sotodlib.io.imprinter_utils as utils
 
+import multiprocessing
+from concurrent.futures import ProcessPoolExecutor, as_completed
+
+
+def bind_books_parallel(platform, book_list, n_proc ):
+    imprint = Imprinter.for_platform(platform)
+    session = imprint.get_session()
+    bid_list = [book.bid for book in book_list]
+    failed_list = []
+    with ProcessPoolExecutor(n_proc) as exe:
+        futures = [
+            exe.submit(_bookbinding_helper, platform, bid) for bid in bid_list
+        ]
+        for future in as_completed(futures):
+            bid, status, message, _  = future.result()
+            imprint.logger.info(f"Just finished book {bid}")
+            book = session.query(Books).filter(Books.bid == bid).one()
+            book.status = status
+            book.message = message
+            session.commit()
+            if status == FAILED:
+                failed_list.append( book.bid)
+    return failed_list
+
+def _bookbinding_helper(platform, bid ):
+    imprint = Imprinter.for_platform(platform)
+    return imprint._run_book_binding(bid)
+
 
-def main(config: str):
+def main(config: str, parallel_oper:bool=False):
```

Comment:
Instead of parallel_oper, just take n_proc?  Then if it it's more than 1, trigger parallel behavior.